core/modules/startup/render.js

```diff
@@ -51,9 +51,11 @@ exports.startup = function() {
 	$tw.styleElement.innerHTML = $tw.styleWidgetNode.assignedStyles;
 	document.head.insertBefore($tw.styleElement,document.head.firstChild);
 	// Prepare refresh mechanism
-	var deferredChanges = Object.create(null),
-		timerId,
-		throttledRefreshFn = function(changes,callback,mainCondition,styleCondition) {
+	var mainDeferredChanges = Object.create(null),
+		styleDeferredChanges = Object.create(null),
+		mainTimerId,
+		styleTimerId,
+		throttledRefreshFn = function(changes,deferredChanges,timerId,throttledRefresh,callback,mainCondition,styleCondition) {
```

Comment:
I think it would be better to add the new function parameters as an options-object. So the function can be extended without changing the function signature again. The more parameters a function has the more complicated it is to call it. 

So imo you should go with

```
throttledRefreshFn = function(changes,callback,mainCondition,styleCondition,options) {
options = options || {}
...
```

Reply:
Thanks @pmario ! Will do!

Reply:
@pmario - I've moved almost everything into the options object